src/Handler/Feed.hs

```diff
@@ -0,0 +1,26 @@
+module Handler.Feed
+    ( getFeedR
+    ) where
+
+import Text.Hamlet (hamletFile)
+
+import Import
+import Query
+import Helper
+
+getFeedR :: Handler Html
+getFeedR = do
+    screams' <- runDB $ recentScreams
+    images <- runDB $ fetchImagesForScreams screams'
+    feedFromScreams $ map (joinOneToMany screamImage images) screams'
+
+feedFromScreams :: [(Entity Scream, [Entity Image])] -> Handler Html
+feedFromScreams screams = do
+    render <- getUrlRender
+    pc <- widgetToPageContent $ do
```

Comment:
do you need `do` here? is there only one line below or should that second line be indented?